addons/barcodes_gs1_nomenclature/data/barcodes_gs1_rules.xml

```diff
@@ -334,5 +334,873 @@
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+        </record>
+
+        <record id="barcode_rule_gs1_350y_and_from_352y_to_356y" model="barcode.rule">
+            <field name="name">Unused GS1 AI measure rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3500</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(35[02-6][0-5])(\d{6})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="barcode_rule_gs1_362y_363y_and_from_366y_to_369y" model="barcode.rule">
+            <field name="name">Unused GS1 AI measure rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3620</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(36[236-9][0-5])(\d{6})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="barcode_rule_gs1_390y_392y" model="barcode.rule">
+            <field name="name">Unused GS1 AI applicable amount payable rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3900</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(39[02][0-9])(\d{0,15})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="barcode_rule_gs1_391y_393y" model="barcode.rule">
+            <field name="name">Unused GS1 AI applicable amount payable rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3910</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(39[13][0-9])(\d{3}\d{0,15})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
```

Comment:
```suggestion
            <field name="name">Amount payable and ISO currency code</field>
            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
            <field name="sequence">3910</field>
            <field name="encoding">gs1-128</field>
            <field name="pattern">(39[13][0-9])(\d{3}\d{0,15})</field>
            <field name="type">skip</field>
            <field name="gs1_content_type">identifier</field>
            <field name="gs1_decimal_usage">True</field>
```

I think it's type `identifier` because the d{3} part will be currency codes (i.e. alphabet chars)